docs/UserGuide.md

```diff
@@ -43,144 +42,260 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
+Adds a task to Dueue<br>
+Format: `add n/TASKNAME [due/DUEDATE] [t/TIME] [#LISTNAME] [d/DESCRIPTION] [@VENUE] [p/PRIORITYLEVEL] [*f]`
 
-> Persons can have any number of tags (including 0)
+> * Tasks cannot have multiple lists.
+> * Tasks must be added with task name.
+> * The required field for a task is TASKNAME, while other fields are optional.
+> * By default, task will be added to current viewing list, unless [#LISTNAME] is specified.
+> * Due dates of tasks can repeating, e.g. every sunday.
+> * Time of each task can only be one time point, e.g. 4pm.
+> * Specifications of fields for task can be entered in any order.
+> * If no fields are entered, help message for add command will appear.
+> * User can choose to replace the priority levels `IMPORTANT`, `NORMAL` and `TRIVIAL` by 3, 2 and 1.
 
 Examples:
 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe t/friend e/betsycrowe@gmail.com a/Newgate Prison p/1234567 t/criminal`
+* `add n/laundry due/every sunday #personal d/wash clothes @B1 *f`<br>
+  Add a task with taskname `laundry` due `every sunday` under the list `personal` with description `wash clothes` at the venue `B1` and mark it as favourite.
+* `add n/2103lecture due/every friday t/4pm #study d/go for lecture p/3 @iCube`<br>
+  Add a task with taskname `2103lecture` due `every friday` at time `4pm` under the list `study` with description `go for lecture` at the venue `iCube` with priority level `IMPORTANT`
+
+### 2.3. Listing all tasks : `list`
 
-### 2.3. Listing all persons : `list`
+Shows a list of all tasks in Dueue in a specific list and/or favourite and switch to that list and/or favourite<br>
+Format: `list [favourite]`<br>
+Format: `list list LISTINDEX [favourite]`<br>
+Format: `list list LISTNAME [favourite]`
 
-Shows a list of all persons in the address book.<br>
+Shows all unfinished tasks in Dueue and unswitch<br>
 Format: `list`
 
-### 2.4. Editing a person : `edit`
+Shows all tasks in Dueue (finished and unfinished) and unswitch<br>
+Format: `list all`
+
+Shows a list of all list names (together with list indices) in Dueue and switch to list of lists<br>
+Format: `list lists`
+
+> * Do not need to specify whether the parameter is LISTINDEX or LISTNAME
+> * Returns only favorite tasks by adding keyword "favorite"
+> * Definition of **switch** : all the tasks added after switching will be automatically added to the current list (favourite) unless otherwise stated
+> * Definition of **unswitch** : switch back to main, the tasks added will not be automatically added to this list (favourite)
+
+Examples:
+
+* `list`<br>
+  Lists all unfinished tasks
+* `list all`<br>
+  Lists all unfinished and finished tasks
+* `list work`<br>
+  List all unfinished tasks in list `work`
+* `list study favourite`<br>
+  List all favourite tasks in list `study`
+* `list lists`<br>
+  List all listname with list indices
+
+### 2.4. Adding a new list : `add list`
```

Comment:
- Suggestion: Maybe it is easier for the user to remember the commands if you group them and start each command with the category name. e.g. all list related command start with "list", e.g. "list add", "list update", etc.